### ISSUE TITLE
[Fluxstore - Woo] -  Categories are not loaded into the application.

### ISSUE THREAD

**Issue by sketchuprzr** [inspireui/support#1863]
1. My issue:

Hi ,

1- only 4 categories appear in the application. How do I make all the categories appear?

My Website: https://www.pratikbakkal.com

Thank you.

2. Screenshots issues:

1)

![image](https://user-images.githubusercontent.com/60021696/72657782-ac429880-39b9-11ea-9c6c-e9a9e0f81e7d.png)

3. Proof of my purchased:

![image](https://user-images.githubusercontent.com/60021696/72657795-e318ae80-39b9-11ea-82aa-a7efa564c688.png)

3. Config files (optional)

**Comment by minhcasi** [comment]
@sketchuprzr - it will load the parent category that have id=0 and child of this category - other sub of subcategories will not be display

**Comment by sketchuprzr** [comment]
> @sketchuprzr - it will load the parent category that have id=0 and child of this category - other sub of subcategories will not be display

There are 10 main categories in my website. I want these 10 categories to be displayed. The id of all categories is different from 0. Why only 4 categories appear first? I want to fix this. What is the point of the program if other categories will not be visible?

**Comment by sketchuprzr** [comment]
hi,
unfortunately my problem is not solved. The contents I have added to the categories are still not visible. Category titles starting with A, B, C, D, E, F, G, I appear, but other names are not. Can you explain clearly which category I can interfere with the category page?

**Comment by minhcasi** [comment]
@sketchuprzr - could you share your config file so that we could help for checking - thank you!

**Comment by sketchuprzr** [comment]
Hello, I sent the required files (config etc.)  to the "minhcasi@gmail.com" mail address.

**Comment by sketchuprzr** [comment]
Hello, I'm still waiting for the solution. Please help me.

**Comment by minhcasi** [comment]
@sketchuprzr - pls try to update latest 1.6.3 and check this code for customizing the categories display base on your requirement - https://tppr.me/BOyeT - thank you!

**Comment by sketchuprzr** [comment]
Hello, I upgraded to version 1.6.3 as you said. But the app still doesn't see other categories! It does not attract categories outside of 4 categories!

I did not understand what to change in the config setting you showed in the picture.

![image](https://user-images.githubusercontent.com/60021696/72894034-80ae0e00-3d2b-11ea-85d9-30510bfbbbed.png)

![image](https://user-images.githubusercontent.com/60021696/72894044-84da2b80-3d2b-11ea-9bde-ae2f5c365ec1.png)
